### PR TITLE
Report copy-edit (in progress)

### PR DESCRIPTION
**Pull request by AmeliaBR**
I'm catching up by reading the draft report & copy editing it as I go. Mostly focused on breaking up long sentences/paragraphs, removing overly formal language or jargon, and some re-arrangements for clarity. I've also made some edits specifically to avoid making it seem like the report is on behalf of only one part of the Maps for the Web community (there were a few first-person references to the geospatial standards side of things).

Feel free to merge after review, but otherwise I'll push more commits as I go.

The initial commit contains:

- Copy edits to the introduction and accessibility sections.

- Style rules for block quotes, and changes to the markdown to remove formatting markup from quotes.

- Also normalized editing marks (ellipses and parentheses) for quotes.

### CODE REVIEW

**Comment by tguild** [comment]
I defer largely to Peter. He and I discussed long sentence style which I sometimes do as well but I think only broke apart one or two in earlier round. Agree it will help for non-native speakers especially and you did a great job preserving the points in your restructuring.

**Comment by prushforth** [comment]
I'm still reviewing.  I just noticed that there are two copies of report.md, maybe get rid of the out of date one.

File: report.md [hunk 1]
```diff
@@ -10,97 +10,101 @@ description: >-
 
 <h2 id="executive-summary">Executive Summary</h2>
 
-Geospatial information on the Web is fragmented, inadequately covered by Web standards, and as a result lacking interoperability significantly raising barriers to its effective use. Efforts to standardize geoinformation have focused on established geospatial developers, which thus leaves out most people. An opportunity exists today to standardize maps and spatial information in HTML, lowering barriers for everyone, ranging from web developers, GIS data providers, map solution providers, the disabled community to the commercial stakeholders. Browser makers are more open than ever to incorporating ideas and code from community initiatives. It falls to communities to organize themselves to assert the need for and definition of such standards. The global geospatial information community can and should mobilize to simplify and standardize geoinformation for the Web, as maps in HTML, supporting both geographic and non-georeferenced map rendering and interaction.
```
Comment (prushforth) [inline report.md:13]: Why delete this:

> Browser makers are more open than ever to incorporating ideas and code from community initiatives. It falls to communities to organize themselves to assert the need for and definition of such standards. The global geospatial information community can and should mobilize to simplify and standardize geoinformation for the Web, as maps in HTML, supporting both geographic and non-georeferenced map rendering and interaction.

?  Maybe it went somewhere I haven't noticed yet...

File: report.md [hunk 2]
```diff
@@ -10,97 +10,101 @@ description: >-
...
+The workshop brought together a panel of global experts and researchers in the emerging field of Web map accessibility. They discussed issues relevant to the use of Web maps by people with disabilities and in particular those with visual impairments, and what actions the Web mapping community can collectively take to facilitate access to map content and functionality. Panelists included Doug Schepers of Fizz Studio, Brandon Biggs an engineer with the Smith Kettlewell Eye Research Institute, Dr. Tony Stockman from Queen Mary University and Dr. Nicholas Giudice from the University of Maine.
```
Comment (prushforth) [inline report.md:57]: empanelling is a bit like drywall, just stickier

**Review by prushforth — APPROVED** [review]
Looks good to me.  A bit sad that we can't have some summary of openness on the part of browser makers, because this is a question that I get repeatedly.  I wanted to highlight the idea that while browsers are open source projects, the browser makers may not feel obliged to pay for inclusions, but they can't prevent them if they are properly founded and funded.

File: report.md (hunk 1 above)
Comment (AmeliaBR) [reply report.md:13]: It's still there, somewhat tempered, in the next paragraph:

> Browser makers are more open than ever to incorporating ideas and code when the web community establishes clear demand for a particular solution.

Comment (AmeliaBR) [reply report.md:13]: The diff isn't really very helpful at tracking changes here.

**Pull request merged**